lib/livebook/hubs.ex

```diff
@@ -31,8 +37,8 @@ defmodule Livebook.Hubs do
 
   Raises `Livebook.Storage.NotFoundError` if the hub does not exist.
   """
-  @spec fetch_hub!(String.t()) :: Provider.t()
-  def fetch_hub!(id) do
```

Comment:
This one makes sense, the raising version is usually `fetch_*!`, as an alternative to `{:ok, _} | :error` :)